upgrades/upgrade_testing_framework/steps/step_select_expectations.py

```diff
@@ -7,11 +11,22 @@ class SelectExpectations(FrameworkStep):
 
     def _run(self):
         # Get the state we need
+        source_version = self.state.test_config.clusters_def.source.engine_version
+        target_version = self.state.test_config.clusters_def.target.engine_version
+
+        path_to_knowledge_base = Path("../knowledge_base")
```

Comment:
This relative path is also hardcoded in the ReportResults, which means we should move it to a centralized location.  Maybe the WorkspaceWrangler?

Reply:
Actually decided to have this be passed down from the main() body as an argument from the command-line.